scripts/get-pr-feedback.sh

```
@@ -0,0 +1,133 @@
+printf "\n"
+printf "  PR #%s · %s · %s unresolved\n" "$PR_NUMBER" "$REVIEW_DECISION" "$UNRESOLVED_COUNT"
+printf "\n"
+
+if [[ "$UNRESOLVED_COUNT" -eq 0 ]]; then
+    printf "  ✓ No unresolved feedback\n\n"
+    exit 0
+fi
+
+# Process each unresolved thread
+echo "$UNRESOLVED_THREADS" | jq -c '.[]' | while read -r thread; do
+    THREAD_ID=$(echo "$thread" | jq -r '.id // ""')
```

Shellcheck (0.11.0)

[warning] THREAD_ID appears unused. Verify use (or export if used externally).

(SC2034)